Dockerfile

```diff
@@ -26,27 +26,32 @@ RUN useradd --create-home --shell /bin/bash $USER && \
     apt-get update && \
     apt-get install -qq --no-install-recommends gettext nginx
 
-# switch to non-root $USER
-USER $USER
-
 # enter app directory
 WORKDIR /home/$USER/app
 
-# update PATH for local pip installs
-ENV PATH "$PATH:/home/$USER/.local/bin"
+# copy config files
+COPY gunicorn.conf.py gunicorn.conf.py
+COPY manage.py manage.py
 
-# install python dependencies
-COPY requirements.txt requirements.txt
-RUN pip install -r requirements.txt
+# overwrite default nginx.conf
+COPY nginx.conf /etc/nginx/nginx.conf
 
 # copy source files
-COPY gunicorn.conf.py gunicorn.conf.py
-COPY manage.py manage.py
 COPY benefits/ benefits/
 COPY bin/ bin/
```

Comment:
Since we're editing this file, want to swap the order of these `COPY` commands?

More likely that `benefits/` will change before `bin/`.